config/go.d/powerdns.conf

```diff
@@ -0,0 +1,168 @@
+# netdata go.d.plugin configuration for powerdns
+#
+# This file is in YAML format. Generally the format is:
+#
+# name: value
+#
+# There are 2 sections:
+#  - GLOBAL
+#  - JOBS
+#
+#
+# [ GLOBAL ]
+# These variables set the defaults for all JOBs, however each JOB may define its own, overriding the defaults.
+#
+# The GLOBAL section format:
+# param1: value1
+# param2: value2
+#
+# Currently supported global parameters:
+#  - update_every
+#    Data collection frequency in seconds. Default: 1.
+#
+#  - autodetection_retry
+#    Re-check interval in seconds. Attempts to start the job are made once every interval.
+#    Zero means not to schedule re-check. Default: 0.
+#
+#  - priority
+#    Priority is the relative priority of the charts as rendered on the web page,
+#    lower numbers make the charts appear before the ones with higher numbers. Default: 70000.
+#
+#
+# [ JOBS ]
+# JOBS allow you to collect values from multiple sources.
+# Each source will have its own set of charts.
+#
+# IMPORTANT:
+#  - Parameter 'name' is mandatory.
+#  - Jobs with the same name are mutually exclusive. Only one of them will be allowed running at any time.
+#
+# This allows autodetection to try several alternatives and pick the one that works.
+# Any number of jobs is supported.
+#
+# The JOBS section format:
+#
+# jobs:
+#   - name: job1
+#     param1: value1
+#     param2: value2
+#
+#   - name: job2
+#     param1: value1
+#     param2: value2
+#
+#   - name: job2
+#     param1: value1
+#
+#
+# [ List of JOB specific parameters ]:
+#  - url
+#    Server URL.
+#    Syntax:
+#      url: http://localhost:80
+#
+#  - api_key
+#    Pre-shared authentication key for access to the REST API.
+#    Syntax:
+#      api_key: secret
+#
+#  - username
+#    Username for basic HTTP authentication.
+#    Syntax:
+#      username: tony
+#
+#  - password
+#    Password for basic HTTP authentication.
+#    Syntax:
+#      password: stark
+#
+#  - proxy_url
+#    Proxy URL.
+#    Syntax:
+#      proxy_url: http://localhost:3128
+#
+#  - proxy_username
+#    Username for proxy basic HTTP authentication.
+#    Syntax:
+#      username: bruce
+#
+#  - proxy_password
+#    Password for proxy basic HTTP authentication.
+#    Syntax:
+#      username: wayne
+#
+#  - timeout
+#    HTTP response timeout.
+#    Syntax:
+#      timeout: 1
+#
+#  - method
+#    HTTP request method.
+#    Syntax:
+#      method: GET
+#
+#  - body
+#    HTTP request method.
```

Comment:
The `body` has the exact same description of `method`.

Reply:
it is copy/paste, we have that typo in a lot of configs, i will fix it in another PR